assets/agenda/utils.ts

```diff
@@ -1023,12 +1024,18 @@ export function formatCoverageDate(coverage: ICoverage) {
         parseDate(coverage.scheduled).format(COVERAGE_DATE_TIME_FORMAT);
 }
 
+export function formattedCoverageStatusText(coverage: ICoverage) {
+    const statusText = getCoverageStatusText(coverage).replace('Coverage ', '');
```

Comment:
This will not work with translated text, `Coverage` is for English only.

IMO a better approach might be to scrap all your changes and place a new if branch [here](https://github.com/superdesk/newsroom-core/blob/develop/assets/agenda/utils.ts#L1036).

```ts
if (coverage.coverage_status !== 'coverage intended') {
    return DRAFT_STATUS_TEXTS[coverage.coverage_status];
} else if (coverage.workflow_status === WORKFLOW_STATUS.DRAFT) {
    ....
}
```

Reason being, if a Coverage is not planned, not intended, on merit/request, then there would be no need to display an expected date/time to the user, because there should not be one for that coverage.

Instead, with the above suggestion, the tooltip would simply state something like `Coverage not planned`

Reply:
yep got it, Thanks for clarification